plugins/home-page/tsconfig.json

```diff
@@ -1,4 +1,7 @@
 {
   "extends": "../../tsconfig.json",
-  "include": ["src"]
+  "include": ["src"],
+  "compilerOptions": {
+    "module": "es6"
+  }
```

Comment:
should be added to the template too

Reply:
Maybe this is unnecessary now that we switched back to commonjs. I'll have a look

Reply:
Still necessary, and it is in the template already.

Reply:
Can we move it to the root config?